armi/bookkeeping/mainInterface.py

```diff
@@ -80,30 +80,49 @@ def _activateDB(self):
             # load case before going forward with normal cycle
             runLog.important("MainInterface loading from DB")
 
-            # Load the database from the point just before start cycle and start node
-            # as the run will continue at the begining of start cycle and start node,
-            # and the database contains the values from the run at the end of the
-            # interface stack, which are what the start start cycle and start node
-            # should begin with.
+            # Load the database from the point just before start cycle and start node as the run
+            # will continue at the begining of start cycle and start node, and the database contains
+            # the values from the run at the end of the interface stack, which are what the start
+            # start cycle and start node should begin with.
 
-            # NOTE: this should be the responsibility of the database, but cannot
-            # because the Database is last in the stack and the MainInterface is
-            # first
+            # NOTE: this should be the responsibility of the database, but cannot because the
+            # database is last in the stack and the MainInterface is first
             dbi.prepRestartRun()
             self.r.p.cycle = self.cs["startCycle"]
             self.r.p.timeNode = self.cs["startNode"]
 
     def _moveFiles(self):
-        # check for orificed flow bounds files. These will often be named based on the
-        # case that this one is dependent upon, but not always. For example, testSassys
-        # is dependent on the safety case but requires physics bounds files.  now copy
-        # the files over
+        """
+        At the start of each run, arbitrary lists of user-defined files can be copied around.
+
+        This logic is controlled by the settings ``CONF_COPY_FILES_FROM`` & ``CONF_COPY_FILES_TO``.
```

Comment:
I'm pretty certain that the capital snake case `CONF` variables will not get rendered as the actual string names, right? In situations like this, I never know if it would be better to use the variable name or the string name. One of the drawbacks of these `CONF` variables, IMO. 

I am not partial to either way, but perhaps consider if it would be better to just use the strings in this docstring instead of the variable names. Up to you.

Reply:
> I'm pretty certain that the capital snake case CONF variables will not get rendered as the actual string names, right?

Nope, in this situation, it will just literally say "CONF_XYZ". I thought that was better, but it's a close call, for me.

I can change it, if you like, sure.

Reply:
Done